README.md

```diff
@@ -213,25 +213,24 @@ to appeal.
 
 ## Fees
 
-The dex does not charge trading fees, but users have to pay on-chain transaction
-fees. Unlike on centralized exchanges where users can trade any arbitrary amount
-of funds, orders on the dex must be made in increments of a specified lot size.
-For example, if the lot size for the DCR/BTC market is 40 DCR, users can
-make orders to buy or sell DCR in increments of 40.
-
-You might be lucky and have your entire order matched by one other user. In
-this case, you will have to pay 2 transaction fees: one for sending funds on
-one chain, and one for receiving funds on the other. In the worst case, each
-lot of your order is matched by a different user, and you will pay a separate
-sending and recieving transaction for each match. Check the 
+The dex does not charge fees, but users pay on-chain transaction fees.
```

Comment:
Actually, "**trading** fees" is better here, since there is still a registration fee.